charts/argo/templates/server-cluster-roles.yaml

```diff
@@ -29,11 +29,20 @@ rules:
   - list
   - watch
   - delete
+- apiGroups:
+  - ""
+  resources:
+  - events
```

Comment:
Do we want to give argo-server the abilty to create/patch events? should it maybe be given readonly access?

Reply:
the change comes from https://github.com/argoproj/argo/pull/3929 - Unsure if this should be included in `argo-helm` by default as it's an optional feature. 
You vote to remove that again, I suppose? That'd be ok for me.